portfolio.html

```diff
@@ -0,0 +1,54 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="./portfolio-style.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="logo-wrapper">
+    <img class="logo" src="./images/acastandardcirclefullname.png" alt="">
+  </div>
+    <nav>
+      <div class="nav-button-wrapper">
+        <a href="./src/wireframes/resume.html"><button class="nav-button resume">Resume</button></a>
```

Comment:
in this `a` tag, add this attribute `target="_blank"`
`<a href="./src/wireframes/resume.html" target="_blank" >`
it'll open the page in a new tab